doc/userguide/rules/payload-keywords.rst

```diff
@@ -280,6 +280,8 @@ bsize
 With the ``bsize`` keyword, you can match on the length of the buffer. This adds
 precision to the content match, previously this could have been done with ``isdataat``.
 
+bsize uses an :ref:`unsigned 64-bits integer <rules-integer-keywords>`.
+
 An optional operator can be specified; if no operator is present, the operator will
 default to '='. When a relational operator is used, e.g., '<', '>' or '<>' (range),
 the bsize value will be compared using the relational operator. Ranges are inclusive.
```

Comment:
Question: considering Jeff's question about range being inclusive or exclusive, and this note here while the new description says ranges are exclusive, are we changing behavior, and do we need to adjust behavior, or just documentation?

Reply:
We are not changing behavior. Ranges are exclusive, and more generally speaking comparisons are strict, except for nfs.procedure and nfs.version which have this historic behavior...

Reply:
Oh, right... So was this bit of explanation wrong?

Reply:
Yes, this doc is wrong (it does not reflect the behavior cf uint.rs
```
        DetectUintMode::DetectUintModeRange => {
            if val > x.arg1 && val < x.arg2 {
                return true;
            }
        }
```

Reply:
ooops 😅